A62-pick-first.md

```diff
@@ -0,0 +1,113 @@
+A62: `pick_first`: sticky TRANSIENT_FAILURE and address order randomization 
+----
+* Author(s): Easwar Swaminathan (@easwars)
+* Approver: @markdroth
+* Status: Draft
+* Implemented in:
+* Last updated: 2023-04-20
+* Discussion at: <google group thread> (filled after thread exists)
+
+## Abstract
+
+This document describes a couple of changes being made to the `pick_first` LB
+policy with regards to
+- Expected behavior when connections to all addresses fail, and
+- Support for address order randomization
+
+## Background
+
+All gRPC implementations contain a simple load balancing policy named
+`pick_first` that can be summarized as follows:
+- It takes a list of addresses from the name resolver and attempts to connect to
+  those addresses one at a time, in order, until it finds one that is reachable.
+  - All RPCs sent on the gRPC channel will be sent to this address.
+- If this connection breaks at a later point in time, `pick_first` will not
+  attempt to reconnect until the application requests that it does so, or makes
+  an RPC.
+- If none of the addresses are reachable, it applies an exponential backoff
+  before attempting to reconnect.
+
+When connections to all addresses fail, there are some similarities and some
+differences between the Java/Go implementations and the C-core implementation.
+
+Similarities include:
+- Reporting `TRANSIENT_FAILURE` as the connectivity state to the channel.
+- Applying exponential backoff. During the backoff period, `wait_for_ready` RPCs
```

Comment:
"wait_for_ready" can be a link to https://github.com/grpc/grpc/blob/master/doc/wait-for-ready.md.